libs/community/langchain_community/embeddings/gpt4all.py

```diff
@@ -19,14 +19,32 @@ class GPT4AllEmbeddings(BaseModel, Embeddings):
 
     client: Any  #: :meta private:
 
+    def __init__(
```

Comment:
whats this needed for?

Reply:
- I am not used to pydantic. Updated the PR to adopt the pydantic `BaseModel` convention and removed the  `__init__`. 
- Edited the docstring based on what i have seen in `huggingface.py`.